### DISCUSSION TITLE
受限网络中内存泄漏

### DISCUSSION THREAD

**Discussion by xmexg** [v2rayA/v2rayA#1492] [General]
未登录校园网前, v2raya会不断发送数据包且不释放内存, 最终耗尽系统内存死机
![image](https://github.com/user-attachments/assets/1c105c33-d542-43a8-b398-aa36c6b05684)

**Comment by Yueyanc** [comment]
同样的问题，内存占用会不断增加

**Comment by mdd3135** [comment]
是的，我也发现了这个现象，卡死后会触发oom-killer，但是杀死了v2raya后又会复活，会不断的卡死，希望可以定位修复
